assets/stylesheets/common/discourse-post-event.scss

```diff
@@ -308,32 +363,56 @@ $interested: #fb985d;
       @include ellipsis;
     }
   }
+}
 
-  .event-dates {
-    // hardcoded as cooking date is async and will change height after initial rendering otherwise
-    // not ideal but a decent low tech solution
-    height: 24px;
-
-    .participants {
-      margin-left: 0.5em;
-      color: var(--primary-medium);
-    }
+.event-dates {
+  // hardcoded as cooking date is async and will change height after initial rendering otherwise
+  // not ideal but a decent low tech solution
+  height: 24px;
 
-    .date {
-      color: var(--primary-high);
-      margin-left: 1em;
+  .participants {
+    margin-left: 0.5em;
+    color: var(--primary-medium);
+  }
 
-      .discourse-local-date {
-        .d-icon {
-          display: none;
-        }
-      }
+  .discourse-local-date {
+    .d-icon {
+      display: none;
     }
+  }
 
-    .separator {
-      color: var(--primary-high);
-      margin: 0 0.5em;
-      text-align: center;
+  .separator {
+    color: var(--primary-high);
+    margin: 0 0.5em;
+    text-align: center;
+  }
+}
+
+.discourse-post-event {
+  .cooked-date,
+  .participants,
+  .event-url .url,
+  .no-rsvp-description {
+    font-size: 0.875rem;
+    line-height: 1.25rem;
+    letter-spacing: 0.00625rem;
+    font-weight: 400;
+    border-bottom: none;
+    margin: 0;
+  }
+  .event__section.no-rsvp {
+    height: calc(48px - 1em);
```

Comment:
what is the 1em here?

Reply:
This is the 0.5em padding on top and bottom. So the height is taking into consideration the padding that will be added.